### DISCUSSION TITLE
Any way to use via ruby?

### DISCUSSION THREAD

**Discussion by inspire22** [qdrant/fastembed#236] [Ideas]
I'm a bit fan of qdrant via the ruby interface https://github.com/patterns-ai-core/qdrant-ruby, and was hoping to try fastembed, but is it only available in python?  

It'd be pretty slick to have it handled on the qdrant server side, so you just add/query for documents using their original text and it internally uses fastembed to make embeddings.

**Comment by NirantK** [comment]
Hey, we don't have a plan to support Ruby in the short run but will definitely consider it
